src/lib/Projection/tween.ts

```diff
@@ -0,0 +1,5 @@
+import { tweened } from 'svelte/motion';
+
+export const tween = tweened(0, {
+  duration: 1500
+});
```

Comment:
Is there any potential this gets reused? Might make sense to allow the start and end values (at the very least if not any other values that can be passed into this) to be passed in from whatever is calling/using this.